apps/design/frontend/src/contests_screen.tsx

```diff
@@ -121,13 +160,38 @@ function ContestsTab(): JSX.Element | null {
                 value={filterPartyId}
                 onChange={(value) => setFilterPartyId(value ?? FILTER_ALL)}
                 style={{ minWidth: '8rem' }}
+                disabled={isReordering}
               />
             )}
           </React.Fragment>
         )}
+        <div style={{ marginLeft: 'auto' }}>
+          {isReordering ? (
+            <Row style={{ gap: '0.5rem' }}>
+              <Button onPress={() => setReorderedContests(undefined)}>
+                Cancel
+              </Button>
+              <Button
+                onPress={onSaveReorderedContests}
+                variant="primary"
```

Comment:
Rather than doing the `assertDefined` in `onSaveReorderedContests`, you can pass the value into the function (I used the parameter name `newContests`). TypeScript is smart enough to know that `reorderedContests` is not undefined because `isReordering` is truthy in this branch, so this works without any assertions or casting.
```suggestion
                onPress={() => onSaveReorderedContests(reorderedContests)}
```

Reply:
Nice, that's a good idea